MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -18,8 +18,14 @@ public protocol InstructionsBannerViewDelegate: class {
     /**
      Called when the user drags either up or down on the `InstructionsBannerView`.
      */
+    @available(*, deprecated, message: "Please use didSwipeInstructionsBanner instead.")
```

Comment:
This PR causes `didDragInstructionsBanner(_:)` to no longer get called. Deprecating it would be misleading, because that would imply (especially to Objective-C code, via a warning) that it’s OK to keep using the old method for now. Either mark this method `obsoleted` or, preferably, conditionally call the old method if it’s implemented.

Reply:
I am calling `didDragInstructionsBanner(_:)` again